core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/kubernetes/upgrade-master0.yml

```diff
@@ -95,6 +95,14 @@
 - name: k8s/master0 | Wait for cluster's readiness
   include_tasks: utils/wait.yml
 
+# Even though cluster is seen as ready here, the next task may occasionally fail with error like
+# "Error from server (NotFound): the server could not find the requested resource".
+# The following task prevents this issue.
+- name: k8s/master0 | Refresh api-resources cache
+  shell: kubectl api-resources --cached=false > /dev/null
```

Comment:
This is an old workaround we extracted from the upgrade role (older commits) and I can confirm it was giving great results! :heart_eyes: 

@to-bar can you elaborate about the testing method? Was it statistical? How many repetitions? :drooling_face:

Reply:
Two iterations - one for v0.4 and second for v0.5 😉